README.rst

```diff
@@ -1,13 +1,6 @@
 Juju Charm Tools
 ================
 
-|travis| |build|
-
-.. |travis| image:: https://travis-ci.org/juju/charm-tools.svg
-    :target: https://travis-ci.org/juju/charm-tools
-.. |build| image:: https://build.snapcraft.io/badge/juju/charm-tools.svg
-    :target: https://build.snapcraft.io/user/juju/charm-tools
-
```

Comment:
i think the `build` target could be replaced with:

```
[![charm](https://snapcraft.io/charm/badge.svg)](https://snapcraft.io/charm)
```

e.g. [![charm](https://snapcraft.io/charm/badge.svg)](https://snapcraft.io/charm)

Reply:
Thank you! I was not aware of the new badge/location.